ejmask-api/src/main/java/com/ebay/ejmask/api/IPatternBuilder.java

```diff
@@ -36,6 +40,15 @@ public interface IPatternBuilder {
      */
     String buildReplacement(int visibleCharacters, String... fieldNames);
 
+    /**
+     * Build pattern to match
+     * @param visibleCharacters as no of characters to be visible.
+     * @param fieldNames       as list of field names
+     * @return
```

Comment:
> Warning:  Javadoc Warnings
Warning:  /home/runner/work/ejmask/ejmask/ejmask-api/src/main/java/com/ebay/ejmask/api/IPatternBuilder.java:47: warning: no description for @return
Warning:  * @return

Reply:
will this warning cause build failure?